apps/form-builder/src/app/api/forms/route.ts

```diff
@@ -0,0 +1,22 @@
+import { createForm } from "@/server/actions/forms";
+import { zCreateFormRequest } from "@hack4impact-utk/internal-models";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST (request: NextRequest) { 
+
+    
+    const requestBody = await request.json();
+    const variable = zCreateFormRequest.safeParse(requestBody);
```

Comment:
Could we give this a more descriptive variable name other than `variable`?

Reply:
Done!